integration-tests/testconfig/ccip/ccip.toml

```diff
@@ -1,3 +1,7 @@
+[Common]
+# chainlink node funding in native token
+chainlink_node_funding = 1
```

Comment:
I see in the other toml `chainlink_node_funding = 2` with same comment about native token. What does the number denote exactly in this context?

Reply:
Currently it uses same amount of native to fund chainlink nodes across all network. If you are running this in testnet 1 native is not enough(example 1avax), therefore in the testnet testconfig you might have seen this referred as 2. For simulated geth 1 Eth is enough for the course of our test.